### ISSUE TITLE
🚀 Release v1.59.0

### ISSUE THREAD

**Issue by mrnicegyu11** [ITISFoundation/osparc-simcore#4623]
### Release version

1.59.0

### Commit SHA

`1d1c5d6b727b9fc4d7cd0c246e990c900498922c`

### Previous pre-release

https://github.com/ITISFoundation/osparc-simcore/releases/tag/staging_TheNameless2

### Did the commit CI succeeded?

- [x] The commit CI succeeded.

### Motivation

- Regular release

#### Staging Sundae3
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4580

* https://github.com/ITISFoundation/osparc-simcore/pull/4511 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4534 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4535 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4537 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4499 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4536 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4539 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4533 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4544 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4545 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4547 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4548 by @mrnicegyu11
* https://github.com/ITISFoundation/osparc-simcore/pull/4505 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4549 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4551 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4550 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4542 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4557 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4558 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4553 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4491 by @YuryHrytsuk
* https://github.com/ITISFoundation/osparc-simcore/pull/4560 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4564 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4552 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4563 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4566 by @YuryHrytsuk
* https://github.com/ITISFoundation/osparc-simcore/pull/4502 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4567 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4565 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4569 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4576 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4577 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4582 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4584 by @mrnicegyu11
* https://github.com/ITISFoundation/osparc-simcore/pull/4575 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4585 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4571 by @YuryHrytsuk
* https://github.com/ITISFoundation/osparc-simcore/pull/4586 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4568 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4573 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4589 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4592 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4583 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4591 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4570 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4572 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4600 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4578 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4599 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4605 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4601 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4603 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4606 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4596 by @bisgaard-itis
* https://github.com/ITISFoundation/osparc-simcore/pull/4607 by @odeimaiz

#### Staging Baklava1
**Release Issue**: n/a

* https://github.com/ITISFoundation/osparc-simcore/pull/4609 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4610 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4614 by @odeimaiz

#### Staging Baklava2
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4656

* https://github.com/ITISFoundation/osparc-simcore/pull/4617 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4612 by @bisgaard-itis
* https://github.com/ITISFoundation/osparc-simcore/pull/4621 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4624 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4608 by @dependabot
* https://github.com/ITISFoundation/osparc-simcore/pull/4625 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4613 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4629 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4630 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4627 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4632 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4562 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4635 by @mguidon

#### Staging Baklava3
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4700

* https://github.com/ITISFoundation/osparc-simcore/pull/4646 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4645 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4639 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4647 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4650 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4649 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4611 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4653 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4626 by @bisgaard-itis
* https://github.com/ITISFoundation/osparc-simcore/pull/4648 by @YuryHrytsuk
* https://github.com/ITISFoundation/osparc-simcore/pull/4660 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4663 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4662 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4671 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4672 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4677 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4658 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4679 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4676 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4675 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4681 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4666 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4684 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4682 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4680 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4689 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4686 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4690 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4688 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4685 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4691 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4687 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4694 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4695 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4693 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4698 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4699 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4696 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4692 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4602 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4705 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4706 by @dependabot
* https://github.com/ITISFoundation/osparc-simcore/pull/4710 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4712 by @matusdrobuliak66

#### Staging Baklava4
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4744

* https://github.com/ITISFoundation/osparc-simcore/pull/4683 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4709 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4590 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4716 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4707 by @bisgaard-itis
* https://github.com/ITISFoundation/osparc-simcore/pull/4717 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4704 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4718 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4719 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4720 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4721 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4726 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4723 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4725 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4728 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4655 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4730 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4729 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4732 by @GitHK

#### Staging Baklava5
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4755

* https://github.com/ITISFoundation/osparc-simcore/pull/4702 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4703 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4736 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4715 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4735 by @odeimaiz
* https://github.com/ITISFoundation/osparc-simcore/pull/4742 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4738 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4741 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4746 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4740 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4749 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4753 by @sanderegg

#### Staging TheNameless1
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4772

* https://github.com/ITISFoundation/osparc-simcore/pull/4754 by @bisgaard-itis
* https://github.com/ITISFoundation/osparc-simcore/pull/4747 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4761 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4759 by @odeimaiz

#### Staging TheNameless2
**Release Issue**: https://github.com/ITISFoundation/osparc-simcore/issues/4784

* https://github.com/ITISFoundation/osparc-simcore/pull/4763 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4769 by @dependabot
* https://github.com/ITISFoundation/osparc-simcore/pull/4697 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/4760 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4758 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4762 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/4778 by @pcrespov
* https://github.com/ITISFoundation/osparc-simcore/pull/4771 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4775 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/4783 by @sanderegg

### Devops check 👷

- [x] Add changes (if any) and release tags in https://github.com/ITISFoundation/osparc-ops-environments/tags
- [x] Add changes (if any) and release tags in **osparc-ops-deployment-configuration** (osparc.io & osparc.speag.com & tip.itis.swiss)

####  (⚠️ DEVOPS) PRs
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4548
  * YH: LGTM. It should not affect the prod. @mrnicegyu11, I will leave it to you to "tick out"
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4491
  * All good. The  env on prod and stag looks the same (YH)
  * :warning: WEBSERVER_STATIC env on stag and prod are different. It doesn't look suspicious but jfyi
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4502
  * [ACTION :stop_sign: ] Observe carefully `director-v2` to be green when deploying 
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4567
  * ENV is in repo.config and in ops-env repo. All good (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4571
  * Removes `node.platform.os == linux` deploy label constraints 
  * All good. It should not affect anything on production (YH)
- [x] https://github.com/ITISFoundation/osparc-simcore/pull/4591
  * [ACTION :stop_sign: ] Make manually `replicas: 0` when deploy. It is more of a safety measure
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4570
  * (YH) It is good for now. For later: https://github.com/ITISFoundation/osparc-ops-environments/issues/374
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4566
  * Fixes twilio env (e.g. `TWILIO_ACCOUNT_SID` and 2 more)
  * All looks good and env is provided (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4648
  * Make `webserver` consume all email env settings
  * All good. Env is provided in config files (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4676
  * All good. Service is disabled everywhere except Master (YH)
  * Once service is ready, we will properly set up its ENV 
  * Double check that service is off (0 replicas) when deploying
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4683
  * The release can be done w/o this ENV. All good (YH)
  * Issue to create this ENV later: https://github.com/ITISFoundation/osparc-ops-environments/issues/373
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4717
  * It related to config master. The config is set up properly (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4723
  * (YH) It is good for now. For later: https://github.com/ITISFoundation/osparc-ops-environments/issues/374
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4703
  * 2 new ENV is properly set everywhere (YH)
  * [ :stop_sign:  ]clusters_keeper service ENV is still missing on aws prod / dalco prod (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4715
  * (YH) It is good for now. For later: https://github.com/ITISFoundation/osparc-ops-environments/issues/374
- [x] https://github.com/ITISFoundation/osparc-simcore/pull/4738
  * ENV is added to all deployments (YH)
  * [ACTION :stop_sign: ] Check out `CLUSTERS_KEEPER_*` env is deployed properly (no empty values)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4740
  * ENV is set on all deployments (YH)
- [X] https://github.com/ITISFoundation/osparc-simcore/pull/4697
  * All good (YH)

####  (⚠️ DEVOPS) Other planned changes
- [X] https://github.com/ITISFoundation/osparc-ops-environments/issues/320
  * [ACTION :stop_sign:] Make sure number of buffer machines is `1` after deployment
- [X] Check traefik logs error logs in staging
  * https://github.com/ITISFoundation/osparc-ops-environments/issues/366
- [X] Redeploy monitoring service on `dalco prod` (https://github.com/ITISFoundation/osparc-ops-environments/issues/262)
- [X] Repull and redeploy deployment agent on `aws prod`, `dalco prod` and `tip prod`
  * :warning:  Deployment Agent didn't deploy properly. Thus, we went with manual deployment using CI/CD scripts.
- [X] Restart Graylog on `aws prod`, `dalco prod` and `tip prod`
- [x] Double check products settings in the database

### Tests assessment: e2e testing check 🧪

- e2e tests on stagging looking standard nothing unusual (MD 8:30 27.9.)

### Summary 📝

- [x] Prepare release link
```cmd
make release-prod version=1.59.0  git_sha=1d1c5d6b727b9fc4d7cd0c246e990c900498922c
```
- [x] Draft [release changelog](https://github.com/ITISFoundation/osparc-simcore/releases)
- [x] Announce maintenance ( ** ANNOUNCE AT LEAST 24 HOURS BEFORE ** )
- redis ``{"start": "2023-09-28T06:00:00.000Z", "end": "2023-09-28T10:00:00.000Z", "reason": "Release v1.59.0"}``
    - [x] aws
    - [x] dalco
    - [x] tip
- status page (https://manage.statuspage.io/)
    - [x] osparc
    - [x] s4l
- mattermost channels
    - [x] maintenance
    - [x] power users

### Special Maintenance Tasks
# URGENT
- [x] Update lite license server to remove unstructured Solvers @mguidon 
### Releasing 🚀
- [x] Maintenance page up.
```cmd
cd /deployment/production/osparc-ops-environments
make up-maintenance
make down-maintenance
```

- [x] Check hanging sidecars. Helper command to run in director-v2 CLI `simcore-service-director-v2 close-and-save-service <uuid>`
- [x] Release by publishing [draft](https://github.com/ITISFoundation/osparc-simcore/releases)
- [x] Check release CI
- [x] Check deployed
  - [x] aws deploy
  - [ ] dalco deploy
  - [x] tip deploy
- [x] Check testing endpoint ex. `https://testing.osparc.speag.com/`
- [x] Delete announcement
- [x] Check e2e runs
- [x] Announce
``` md
:tada: https://github.com/ITISFoundation/osparc-simcore/releases/tag/v<M.m.0>
```

**Comment by mrnicegyu11** [comment]
Hotfix in a very hacky way was deployed to `tip.itis.swiss` only. The docker image of webserver and static webserver was changed from `release-github-latest` to `hotfix-github-latest`

**Comment by YuryHrytsuk** [comment]
Be aware of removal of `standardworker` node label constraint. From now on, this label constraints should not appear in dy-sidecars and thus there is no need in having this label on docker nodes in cluster

**Comment by YuryHrytsuk** [comment]
When we release to prod, we need to rotate `S3 Access Key` and `postgres_password`.
We also need to restart deployment agent to make sure it uses new secrets (portainer password)

**Comment by matusdrobuliak66** [comment]
@mrnicegyu11 @YuryHrytsuk Please check the error logs of traefik in the AWS Staging;
```
ime="2023-09-27T06:44:29Z" level=error msg="Router defined multiple times with different configurations in [monitoring-prometheuscadvisor-bs0zbli9yi7k3nkk4g7is7067 monitoring-prometheuscatchall-hfodm8skfcrsmgplruv2jbukp]" providerName=docker routerName=prometheus
```
Something is probably misconfigured, I expect the same problem in production with the release, so please make sure it is OK. thanks

**Comment by pcrespov** [comment]
> @mrnicegyu11 @YuryHrytsuk Please check the error logs of traefik in the AWS Staging;
> 
> ```
> ime="2023-09-27T06:44:29Z" level=error msg="Router defined multiple times with different configurations in [monitoring-prometheuscadvisor-bs0zbli9yi7k3nkk4g7is7067 monitoring-prometheuscatchall-hfodm8skfcrsmgplruv2jbukp]" providerName=docker routerName=prometheus
> ```
> 
> Something is probably misconfigured, I expect the same problem in production with the release, so please make sure it is OK. thanks

FYI: this is also reported in https://github.com/ITISFoundation/osparc-ops-environments/issues/366

**Comment by matusdrobuliak66** [comment]
During this release when doing some manual configuration/fixes (because auto-deployment agent was not working properly @YuryHrytsuk @mrnicegyu11 please explain the issues more in detail) by mistake the master branch containers were deployed. This caused that new commits with DB migration got in and therefore we decided we will fully release master.

What we did?
 - released master --> staging [TheNameless3] https://github.com/ITISFoundation/osparc-simcore/issues/4797
 - released staging --> production [1.60.0] https://github.com/ITISFoundation/osparc-simcore/issues/4799

**Comment by mrnicegyu11** [comment]
During the release we faces several issues:
- The deployment agent `0.11.3` somehow mis-composed the env-vars which resulted in novel vars like `RABBIT_SECURE` missing. There will be follow-up talks on how to improve our handling of env-vars.
- We switched to a manual generation of the docker-compose `stack.yml` file, and by human error we accidently deployed `itisfoundation:master-github-latest` docker images. This resulted in an unwanted, not reversible postgresql DB migration via alembic.
- To mitigate the DB migration, we decided to release `v1.60.0` as well, from the HEAD of `osparc/simcore master`. We deployed this using `docker stack deploy` with a manually generated docker-compose `stack.yml` file. We note that updating existing docker services lead to missing env-vars, but a full `docker stack rm` and subsequent `docker stack deploy` fixed the issue.
